bundles/net.tourbook/src/net/tourbook/Messages.java

```diff
@@ -2548,6 +2548,9 @@ public class Messages extends NLS {
    public static String        Slideout_TourLogOptions_Radio_OpenLog_Never;
    public static String        Slideout_TourLogOptions_Radio_OpenLog_SelectedEvents;
 
+   public static String        Slideout_TourBlogOptions_Checkbox_ShowTourTags;
+   public static String        Slideout_TourBlogOptions_Checkbox_ShowTourTags_Tooltip;
+
    public static String        Slideout_TourMarkerFilter_Checkbox_IsReduceLatLonDigits;
```

Comment:
Should be sorted alphabetically, it is

...TourLog...
...TourBlog...
...TourMarker...

Reply:
Sometimes I do the microsorting not alphabetically, e.g. I prefer

...._Start
...._End

...._On
...._Off

instead of

...._End
...._Start

...._Off
...._On

Reply:
Fixed